Python/sysmodule.c

```diff
@@ -2861,11 +2862,11 @@ PyAPI_FUNC(void)
 PySys_AddWarnOptionUnicode(PyObject *option)
 {
     PyThreadState *tstate = _PyThreadState_GET();
+    _Py_EnsureTstateNotNULL(tstate);
```

Comment:
You can remove these asserts and instead change `_PyThreadState_GET` to `PyThreadState_Get`.

Reply:
I have been asked to add these assert before. @ZeroIntensity , would you mind if we did as @kumaraditya303 suggests?